src/editor/codeeditor.cpp

```diff
@@ -130,6 +131,26 @@ void CodeEditor::setIdeType(const QString &ideType)
     setTextColors();
 }
 
+void CodeEditor::runHighlighterWithDefinition(CodeEditor *sourceDocument)
+{
+    auto cursor = sourceDocument->textCursor();
+    cursor.movePosition(QTextCursor::Start);
+    sourceDocument->setTextCursor(cursor);
+    int linescount = 1;
+    cursor.movePosition(QTextCursor::EndOfLine);
+    constexpr int cFive = 5;
```

Comment:
rename to "veryImportantVariable" and add commend "Don't touch it". cFive is not the best name))

Reply:
but I'm glad to see that you used consexpr. How many Mr.Mayers' books have you read?